### PR TITLE
Secret store changes

### PR DESCRIPTION
**Pull request by adityadani**
Add IBM Key Protect doc 
- Remove the steps for updating config.json in k8s secrets doc.

Modify the Secrets Store Management landing page.
- Move each secret store to its own directory and an index.md file.
- Add logos for Vault, DCOS, Kubernetes
- Show the logos on the main landing page.

### CODE REVIEW

**Comment by adityadani** [comment]
screen shot 2018-12-05 at 6 01 55 pm

**Comment by adityadani** [comment]
The icons are misaligned due to different sizes. I could use some help in fixing those.

**Comment by harsh-px** [comment]
@aaroncavano can you help out here? The icons are misaligned due to different sizes.

File: content/key-management/aws-kms/_index.md [hunk 1]
```diff
@@ -16,7 +18,7 @@ Supported from PX Enterprise 1.4 onwards
 ### Deploying Portworx {#deploying-portworx}
 
 There are multiple ways in which you can setup Portworx so that it gets authenticated with AWS
-
+∏
```
Comment (harsh-px) [inline content/key-management/aws-kms/_index.md]: Is this character intended?

Comment (adityadani) [reply content/key-management/aws-kms/_index.md]: no it is not. I am not sure how it got added. Will remove it.

**Pull request merged**